tests/by-util/test_cp.rs

```diff
@@ -4212,7 +4212,7 @@ fn test_cp_debug_default_sparse_virtual_file() {
     let ts = TestScenario::new(util_name!());
     ts.ucmd()
         .arg("--debug")
-        .arg("/sys/kernel/address_bits")
+        .arg("/sys/kernel/profiling")
```

Comment:
```suggestion
        .arg("/sys/kernel/profiling") // See test_cp_default_virtual_file
```

Reply:
I just ended up adding the comment you suggested in discord in each of the functions.